bundles/org.openhab.binding.deconz/src/main/java/org/openhab/binding/deconz/internal/handler/LightThingHandler.java

```diff
@@ -395,7 +395,14 @@ private void valueUpdated(ChannelUID channelUID, LightState newState) {
                     updateState(channelUID, new DecimalType(miredToKelvin(ct)));
                 }
             }
-            case CHANNEL_POSITION -> updatePercentTypeChannel(channelUID, newState.bri, true); // always post value
+            case CHANNEL_POSITION -> {
+                Integer lift = newState.lift;
+                if (lift != null && lift >= PercentType.ZERO.intValue() && lift <= PercentType.HUNDRED.intValue()) {
+                    updateState(channelUID, new PercentType(lift));
+                } else {
+                    updatePercentTypeChannel(channelUID, newState.bri, true); // always post value
+                }
```

Comment:
I wonder if this is necessary. Is there really any deconz firmware that accepts `lift` as command but does not report the status in `lift`? Since we only set `lift` for the command, I believe it's enough to consider `lift`. I think you can also omit the range check.

Reply:
I tried a one year old deconz firmware and `lift` was already included.
Therefore I modified my change according your suggestion.